libs/sdk-backend-spi/src/workspace/dashboards/dashboard.ts

```diff
@@ -189,6 +189,11 @@ export interface IListedDashboard {
      * Updated date
      */
     readonly updated: string;
+
+    /**
+     * Dashboard (optional) tagging system
```

Comment:
I wonder if we should mention here that the optionality is for backward compatibility purposes and that the caller can rely that tags will be there.. and be at worst empty array?